.yamato/config/__shared.metafile

```diff
@@ -138,6 +138,8 @@ test_platforms:
       - [all]: --scripting-backend=<SCRIPTING_BACKEND> 
       - [all]: --platform=editmode
       - [all]: --extra-editor-arg="-colorspace=<COLORSPACE>"
+      - [Android_OpenGLES3, Android_Vulkan]: --platform=Android
```

Comment:
I was wondering, if this is performance specific stuff, not general editmode (since in general we dont do editmode for those anyways), should it be in performance metafile or shared? I don't have much preference in this, so feel free to leave as is! just thought to bring it to discussion :)

Reply:
i was trying to keep it consistent with standalone and playmode, and ios/android only run performance editmode tests, so this part doesnt affect the other jobs :)